snap/snapcraft.yaml

```diff
@@ -18,7 +18,7 @@ apps:
       PATH: "/snap/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
       # https://github.com/lxc/pylxd/pull/361
       PYLXD_WARNINGS: "none"
-    command: bin/snapcraft
+    command: usr/bin/python3 $SNAP/bin/snapcraft
```

Comment:
just keep this :-)

Reply:
I moved it into its own PR. I'll close this.